static/js/stream_edit.js

```diff
@@ -40,6 +52,12 @@ exports.get_email_of_subscribers = function (subscribers) {
     });
 };
 
+exports.get_person_from_subscribers = function (subscribers) {
+    return subscribers.map(function (user_id) {
+        return people.get_by_user_id(user_id);
+    });
+};
```

Comment:
The function just above this, which this replaces, became unused with this change and should have been deleted.